pkg/irreparabledb/irreparabledb.go

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparabledb
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/migrate"
+	"storj.io/storj/pkg/auth"
+	dbx "storj.io/storj/pkg/irreparabledb/dbx"
+	"storj.io/storj/pkg/pb"
+	pdbauth "storj.io/storj/pkg/pointerdb/auth"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+// Server implements the statdb RPC service
+type Server struct {
```

Comment:
I'm not following why this server is needed?

Reply:
Made changes. Application (checker.go) calls directly the DB access calls. The server structure is to provide the db connection attributes to the application to read/write/delete/update the db entries